ansible/create_admin_user.py

```diff
@@ -0,0 +1,32 @@
+"""
+Creates a new admin user. 
+
+Usage: python create_admin_user.py <Galaxy_API_key> <new_username> <new_user_email> <new_password>
+"""
+from __future__ import print_function
+
+import sys
+
+import bioblend.galaxy
+
+if len(sys.argv) != 5:
+    print("Usage: python create_admin_user.py <Galaxy_API_key> <new_username> <new_user_email> <new_password>")
+    sys.exit(1)
+
+galaxy_url = "http://127.0.0.1:8080"
+galaxy_api_key = sys.argv[1]
+username = sys.argv[2]
+email = sys.argv[3]
+password = sys.argv[4]
+
+if len(password) < 6:
+    print("Pasword for admin user on Galaxy needs to be of at least 6 characters")
+    print("User not added!!!")
+    exit(1)
+
+# Initiating Galaxy connection
+gi = bioblend.galaxy.GalaxyInstance(galaxy_url, galaxy_api_key)
+
+# Create a new user
+new_user = gi.users.create_local_user(username, email, password)
```

Comment:
So, I did a few tests with Bioblend 0.8.0 and it appears the `create_local_user` call shouldn't fail silently.

I found three cases.

1. Fails to connect: raises a request.RequestException -> IOError.  The exception type isn't documented so it's probably safest to catch `StandardError`.
2. Connects but Galaxy gives an error: raises a bioblend.ConnectionError
3. Success:  returns a dictionary with some info about the user.  Here's an example
```{u'email': u'pireddu@crs4.it',
 u'id': u'f597429621d6eb2b',
 u'model_class': u'User',
 u'nice_total_disk_usage': u'0 bytes',
 u'total_disk_usage': 0.0,
 u'username': u'luca'}
```
Rather than depending on the exceptions, we could just print `new_user['id']`.  If bioblend doesn't raise we'll at least have the access to the dictionary raise an exception.